docs/userGuide/components/panel.md

```diff
@@ -141,20 +181,18 @@
 </tip-box>
 <br>
 
-#### If `src` attribute is provided, the panel content is loaded dynamically from the `src` specified. To load it immediately, you can provide the `load-all` attribute. 
+#### If `src` attribute is provided, the panel body will be appended content from the `src` specified.
```

Comment:
"appended" sounds like a jargon-ish word.

If we just want to avoid saying "loaded dynamically", maybe we can use a 'softer' word "taken":

`... the panel content will be taken from...`.